components/DotcmsImage.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
should this be moved in the `Shared` folder?

Reply:
Ohhh I forgot about the `Shared` folder I can put this one and the `DateText` there.